CMakeLists.txt

```
@@ -29,7 +29,7 @@ endif()
 
 project(microxrcedds VERSION "2.0.0")
 
-set(_client_tag develop)
+set(_client_tag feature/update_bin_api)
```

```suggestion
set(_client_tag develop)
```

When https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/294/files merged